lib/schema.js

```diff
@@ -129,6 +129,48 @@ const linkSchemas = (schemas, categories, domains) => {
   return errors.length ? new MetaschemaError(errors) : null;
 };
 
+const extractDefinition = (definition, path, ms) => {
+  const separatorIndex = path.indexOf('.');
+  return separatorIndex === -1 ?
+    definition[path] :
+    extractDefinition(
+      ms.categories.get(
+        definition[path.slice(0, separatorIndex)].category
+      ).definition,
+      path.slice(separatorIndex + 1),
+      ms
+    );
+};
+
+const processView = (view, ms) => {
+  if (view.definition) {
+    return;
+  }
+  view.definition = {};
+  for (const field of view.Fields) {
+    const props = typeof field === 'string' ? { [field]: field } : field;
+    for (const key in props) {
```

Comment:
Nit: perhaps `Object.keys(props)[0]` to make sure we only support one key there or signify that there shouldn't be more than one field.

Reply:
This is intentionally implemented this way to support:
```js
View({
  Category: 'Name',
  Fields: [{
    A: 'B',
    B: 'A',
  }],
})
```